### PR TITLE
Chapter_9: Advanced login

### PR DESCRIPTION
**Pull request by ducminh98-dev**

### CODE REVIEW

**Comment by ducminh98-dev** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -236,3 +236,17 @@ input {
...
+  span {
+    margin-left: 20px;
+    font-weight: normal;
+  }
+}
+
+#session_remember_me {
+  width: auto;
+  margin-left: 0;
+}
```
Comment (thaodtd-0665) [inline app/assets/stylesheets/custom.scss]: end line

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -6,10 +6,14 @@
...
       <%= f.label :email %>
-      <%= f.email_field :email, class: "form-control" %>
+      <%= f.email_field :email, class: 'form-control' %>
 
       <%= f.label :password %>
-      <%= f.password_field :password, class: "form-control" %>
+      <%= f.password_field :password, class: 'form-control' %>
+      <%= f.label :remember_me, class: "checkbox inline" do %>
+      <%= f.check_box :remember_me %>
+      <span>Remember me on this computer</span>
```
Comment (thaodtd-0665) [inline app/views/sessions/new.html.erb]: i18n

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -6,10 +6,14 @@
     <%= form_for(:session, url: login_path) do |f| %>
 
       <%= f.label :email %>
-      <%= f.email_field :email, class: "form-control" %>
+      <%= f.email_field :email, class: 'form-control' %>
```
Comment (thaodtd-0665) [inline app/views/sessions/new.html.erb]: nháy "

**Comment by ducminh98-dev** [comment]
ready

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**